src/app/store/Config/Config.dispatcher.js

```diff
@@ -25,6 +25,7 @@ export class ConfigDispatcher extends QueryDispatcher {
         if (data) {
             BrowserDatabase.setItem(data, 'config', ONE_MONTH_IN_SECONDS);
             dispatch(updateConfig(data));
+            dispatch(setLoadingStatus(false));
```

Comment:
Even knowing that this is redundant (isLoading part), why to create a separate action for this? Just set the loading to false when updaing the config.